plugins/auth-keycloak/src/templates/keycloak.strategy.base.template.ts

```diff
@@ -0,0 +1,33 @@
+import { ConfigService } from "@nestjs/config";
+import { PassportStrategy } from "@nestjs/passport";
+import { KeycloakStrategy } from "@exlinc/keycloak-passport";
```

Comment:
Not added in created package.json.

Reply:
Also, this is creating multiple problems as it doesn't have types and needs a declaration file.

Reply:
![image](https://github.com/amplication/plugins/assets/100484401/1a34aa8d-8e72-43e4-9262-0f7475fe7186)
Some fields like authorizationURL, tokenURL are missing. Could you look into this?